embuilder.py

```diff
@@ -54,11 +54,13 @@
     'gl-webgl2',
     'html5',
     'libc',
+    'libc-sockets',
```

Comment:
the name may be misleading, since it's not the normal sockets code. how about `sockets-proxy` or `sockets-testharness` or such?

Reply:
The library `libc-sockets` does contain the normal sockets code from musl. These were split off to a separate block so that they can be omitted from the build when the proxy code is used. The proxy sockets code is contained in library `libc-sockets-proxy`. (Renamed to this from `libposix-sockets-proxy` for consistency) If proxy code is enabled, `libc-sockets-proxy` is used, otherwise `libc-sockets` is used.